api_docs/swagger/components/requestBodies/partner_order_payload.yaml

```diff
@@ -0,0 +1,24 @@
+type: object
+properties:
+  partner_order:
+    type: object
+    required: true
+    properties:
+      oem_token:
+        required: true
+        type: string
+        example: 987654
+      purchased_at:
+        required: true
+        type: string
+        format: date-time
+        example: "2016-07-06T08:48:22.000Z"
+      email:
+        type: string
+        format: email
+        example: john.doe@example.com
+      partner_order_items:
+        required: true
+        type: array
+        items:
+          $ref: '/swagger/swagger.yaml#/components/requestBodies/PartnerOrderItemPayload'
```

Comment:
In general, I miss the `description` of these fields.
What are they for? Do they need to follow a format? Do they have any validations?
And also importantly: what does SCC do with the data?

For example:
- Does OEM token need to be unique?
  In the case of in-activated subs, we could document that end-users need to use the token to claim their subscriptions; in the case of pre-activated subs, end-users don't need it, but we should state that the renewal order needs to have the same token as the original order, etc.
- Email: do we send the user something? Is only 1 email allowed?
- starts at: how does the end date get calculated? does it really need to be a `date-time`, or can we allow a simple date?